app/controllers/admin/reports_controller.rb

```diff
@@ -57,7 +57,9 @@ def render_data?
 
     def render_report_as(format)
       if OpenFoodNetwork::FeatureToggle.enabled?(:background_reports, spree_current_user)
-        blob = ReportBlob.create_for_upload_later!
+        filename = report_filename
+        filename = "#{filename}html" if report_format.blank?
```

Comment:
I'm wondering if this needs a `.`?
```suggestion
        filename = "#{filename}.html" if report_format.blank?
```

Reply:
Ah, good question. I can see it doesn't, because `report_filename` includes `report_format` [like so](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/controllers/concerns/reports_actions.rb#L58-L64):

    "#{report_type || action_name}_#{file_timestamp}.#{report_format}"

So when blank, it leaves a trailing dot. I wonder if this problem should be solved inside `ReportsActions`. 
Hmm.. yes there is already an assumption made on `report_format.present?`, which passes in `:html` as the format in this method. 

So, I suggest a change to `ReportsActions::report_filename` to accept the format as a parameter:

```suggestion
        filename = report_filename(format)
```

Reply:
I'll have a look. I made this quick fix after finding that a lot more changes were needed to set the format to `:html` by default but your suggestion seems simpler.